"""Holds logic for core errors."""


class ValidationError(Exception):
    """An error that occurred during validation."""

    pass
